server/routes/userRoutes.js

```diff
@@ -95,29 +84,71 @@ module.exports = (db) => {
     dbHelper
       .getBlacklistedSitesWithUserID(userId)
       .then((blacklists) => res.json(blacklists))
-      .catch((err) => res.json(err));
+      .catch((err) => res.status(400).json(err));
   });
 
+  // When a user wants to add a site to their blacklist
   router.post("/blacklists", (req, res) => {
     const userId = req.session.userId;
     if (!userId) {
       return res.status(403).send("A user must be signed in!");
     }
     const { host_name } = req.body.params;
```

Comment:
Should just be `req.body`. `.param`s is for extracting `:variables` from the url

Reply:
Good catch! Fixed error.